.github/workflows/build-images-action.yml

```diff
@@ -4,6 +4,8 @@ on:
     branches:
       - 'main'
```

Comment:
Further confusion about using or not using single quotes. Here the main is also quoted, while most other workflow files it is not.

Reply:
This was added after Kashif commented that it needs a quote, but I later found out that the quote is not needed in GH actions doc.

Reply:
Thanks @smoshiur1237 . Anw after all I decided to add quotes everywhere for the shake of consistency and to prevent future confusions.